docs/resources/user_namespace_access.md

```diff
@@ -0,0 +1,100 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "temporalcloud_user_namespace_access Resource - terraform-provider-temporalcloud"
+subcategory: ""
+description: |-
+  
+---
+
+# temporalcloud_user_namespace_access (Resource)
+
+
+
+## Example Usage
+
+```terraform
+terraform {
+  required_providers {
+    temporalcloud = {
+      source = "temporalio/temporalcloud"
+    }
+  }
+}
+
+provider "temporalcloud" {
+}
+
+resource "temporalcloud_namespace" "terraform" {
+  name               = "terraform-users"
+  regions            = ["aws-us-east-1"]
+  accepted_client_ca = base64encode(file("${path.module}/ca.pem"))
+  retention_days     = 14
+}
+
+resource "temporalcloud_namespace" "second_ns" {
+  name               = "terraform-users-2"
+  regions            = ["aws-us-east-1"]
+  accepted_client_ca = base64encode(file("${path.module}/ca.pem"))
+  retention_days     = 14
+}
+
+resource "temporalcloud_user" "namespace_admin" {
+  email          = "ns_admin@foobar.example"
+  account_access = "developer"
+}
+
+resource "temporalcloud_user" "namespace_write" {
+  email          = "ns_write@foobar.example"
+  account_access = "developer"
+}
+
+resource "temporalcloud_user" "namespace_read" {
+  email          = "ns_read@foobar.example"
+  account_access = "developer"
+}
+
+resource "temporalcloud_user_namespace_access" "admin" {
+  user_id      = temporalcloud_user.namespace_admin.id
+  namespace_id = temporalcloud_namespace.terraform.id
+  permission   = "admin"
+}
+
+resource "temporalcloud_user_namespace_access" "write" {
+  user_id      = temporalcloud_user.namespace_write.id
+  namespace_id = temporalcloud_namespace.terraform.id
+  permission   = "write"
+}
+
+resource "temporalcloud_user_namespace_access" "read" {
+  user_id      = temporalcloud_user.namespace_read.id
+  namespace_id = temporalcloud_namespace.terraform.id
+  permission   = "read"
+}
+
+resource "temporalcloud_user_namespace_access" "read_second_ns" {
+  user_id      = temporalcloud_user.namespace_read.id
+  namespace_id = temporalcloud_namespace.second_ns.id
+  permission   = "read"
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `namespace_id` (String) The ID of the namespace to which this user should be given the requested role
+- `permission` (String) The permission to grant the user in the namespace
+- `user_id` (String) The ID of the user to which this namespace access should be granted
```

Comment:
is the user_id the internal id of the user or the email address of the user? If internal ID, recommend this is specified that it's internal to Temporal as this ID is not exposed in the UI and only exposed in response messages to GetUsers, for example. 

For context, I believe we are adding a GetUser by email_adder (externally facing user id) to the api soon.